### ISSUE TITLE
ip Progress Tracker not detecting branch

### ISSUE THREAD

**Issue by FredericChow00** [nus-cs2103-AY2223S2/forum#94]
Hi all,

I took a look at the ip progress tracker and realized that my branch-level-8 was not detected (even though I have already pushed it) although all other branches that i pushed were detected. May I know if what are some possible reasons for this issue? 

I pushed all these branches a few days ago so it should not be because that the list is not updated

**Comment by gremmyz** [comment]
@FredericChow00 I took a look at a similar issue raised earlier (Issue #73), and it could be that your branch-level-8 hasn't been merged with the master branch?

**Comment by clevon-w** [comment]
@gremmyz I am facing the same issue as @FredericChow00 but I did merge my `branch-Level-8` with the `master` branch.
Like @FredericChow00, all other branches were detected except for `branch-Level-8` as shown in the images below.

![image](https://user-images.githubusercontent.com/88370247/215411501-695868ab-639a-4e7e-a146-a0bf3de000b3.png)
<img width="363" alt="image" src="https://user-images.githubusercontent.com/88370247/215411584-bd2d1d74-e99c-4cca-91b5-fc8b39d7d92f.png">

Any help from anyone would be deeply appreciated!

**Comment by damithc** [comment]
@clevon-w the screenshot shows the tag, not the branch. 
As you can see from https://github.com/clevon-w/ip/branches, that branch is missing altogether in your fork.
Perhaps you forgot to push the branch after merging?

**Comment by FredericChow00** [comment]
@damithc Hi Prof, 

![image](https://user-images.githubusercontent.com/108938188/215639122-b8e78176-61f0-49d4-9678-23b823b01962.png)

![image](https://user-images.githubusercontent.com/108938188/215638925-65bcbbfd-aba5-41ea-bd5f-572defa054b7.png)

It seems that for me my brnaches are active in my fork though, although it is still not getting picked up by the website tracker

**Comment by francisyzy** [comment]
I think the grading script marks it as merged if there are no commits ahead of master. Seems like you have 2 commits ahead of master. ![215638925-65bcbbfd-aba5-41ea-bd5f-572defa054b7](https://user-images.githubusercontent.com/24467184/215652553-025c7b8f-812f-436b-9b22-52ac293a579b.png)

When I merged my code using rebase, there were commits ahead of master and my branches were not marked as merged, I had to fiddle around with those commits and force push those branches.
